build.gradle

```diff
@@ -4,12 +4,13 @@ buildscript {
     }
 
     repositories {
-        google()
-        mavenCentral()
         mavenLocal()
+        mavenCentral()
+        google()
     }
     dependencies {
-        classpath 'com.mparticle:android-kit-plugin:' + project.version
```

Comment:
Sorry I missed this one earlier, this line should remain `classpath 'com.mparticle:android-kit-plugin:' + project.version`